cip/1.accepted/CIP2015-08-06-date-time.adoc

```diff
@@ -785,6 +785,12 @@ Does lhs occur after rhs:
     WITH date(...) AS lhs, date(...) AS rhs // or another temporal instant type
     RETURN lhs > rhs
 
+Instant values that occur at the same point in time, but that have a different timezone are not considered equal, and must therefore be ordered in some predictable way.
+Cypher prescribes after the primary order of point in time, instant values be ordered by effective timezone offset, from west (negative offset from UTC) to east (positive offset from UTC).
+This has the effect that times that represent the same point in time will be ordered with the time with the earliest local time first.
```

Comment:
This is not true for offsets greater than +12:00 or smaller than -12:00 (only true for Time, not for DateTime).

Reply:
True. It could be clarified that it has that effect for _DateTime_.